style.css

```diff
@@ -1 +1,215 @@
 /*  Add your styling here */
```

Comment:
Consider putting your CSS files in a separate folder - it's likely you'll use many of them in real projects